### DISCUSSION TITLE
Techempower Framework Benchmarks

### DISCUSSION THREAD

**Discussion by ohenepee** [uNetworking/uWebSockets.js#154] [Q&A] [answered]
@alexhultman Please include this awesome library in the [TF Benchmarks](https://github.com/TechEmpower/FrameworkBenchmarks) as it would first of all provide a greater exposure and also show where it fits among other frameworks. It would be great to see this particular JS variant added instead of the C++ one.

**Comment by ghost** [comment, marked as answer]
This is a common recurring question/request. I don't consider techempower a reliable set of people and I really do not care enough about not being listed
